yarn-project/aztec.js/src/utils/authwit.ts

```diff
@@ -0,0 +1,24 @@
+import { AztecAddress, CircuitsWasm, GeneratorIndex } from '@aztec/circuits.js';
+import { pedersenPlookupCompressWithHashIndex } from '@aztec/circuits.js/barretenberg';
+import { FunctionCall, PackedArguments } from '@aztec/types';
+
+/**
+ * Compute an authentication witness message hash from a caller and a request
+ * H(caller, target, selector, args_hash)
```

Comment:
can you rename `target` -> `target_contract_address`

Reply:
hahaha why? Cos these can be than just contract addresses?

Reply:
caller and target are both addresses, and requests can only really be targeted to addresses so seemed unnecessary.